goth/runner/log_monitor.py

```diff
@@ -178,9 +178,9 @@ def start(self, in_stream: Iterator[bytes]):
 
     async def stop(self) -> None:
         """Stop the monitor."""
-        await super().stop()
         if self._buffer_task:
             self._buffer_task.stop(StopThreadException)
+        await super().stop()
```

Comment:
This change is to fix #468, in which `_buffer_task` tries to add some more events to a monitor that is already stopped.